lib/brains/forms.py

```diff
@@ -99,6 +105,18 @@ class WebhookVerifyForm(forms.Form):
     def braintree_data(self):
         return self.cleaned_data['bt_challenge']
 
+    def clean_bt_challenge(self):
+        res = requests.get(
+            settings.BRAINTREE_PROXY + '/verify',
+            params={'bt_challenge': self.data['bt_challenge']}
```

Comment:
I couldn't remember if its in cleaned_data when its being cleaned_, probably is right?